src/rolch/online_gamlss.py

```diff
@@ -323,19 +291,21 @@ def update_beta(
 
         return beta, beta_path, rss, lambda_max, lambda_path
 
-    def _make_x_gram_list(self, X, w):
-        """Make the Gramian Matrices G = X.T @ W @ GAMMA @ X"""
-        return [init_gram(x, w, self.forget) for x in X]
+    def _make_initial_fitted_values(self, y: np.ndarray) -> np.ndarray:
+        fv = np.stack(
+            [
+                self.distribution.initial_values(y, param=i)
+                for i in range(self.distribution.n_params)
+            ],
+            axis=1,
+        )
+        return fv
 
     def fit(
         self,
+        X: np.ndarray,
         y: np.ndarray,
-        x0: np.ndarray,
-        x1: Optional[np.ndarray] = None,
-        x2: Optional[np.ndarray] = None,
-        x3: Optional[np.ndarray] = None,
         sample_weights: Optional[np.ndarray] = None,
```

Comment:
I would recommend to be consistent with `sklearn` where this arg is called `sample_weight`

Reply:
I renamed that argument. Thank you for the suggestion.